services/app-api/patches/jsdom+26.0.0.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/jsdom/lib/jsdom/living/xhr/XMLHttpRequest-impl.js b/node_modules/jsdom/lib/jsdom/living/xhr/XMLHttpRequest-impl.js
```

Comment:
I hate this whole thing but it fixes the weird jsdom issue 🤷

Reply:
What's the issue being fixed?

Reply:
```
{"errorType":"Runtime.ImportModuleError","errorMessage":"Error: Cannot find module './xhr-sync-worker.js'\nRequire stack:\n- /var/task/index.js\n- /var/runtime/index.mjs","trace":["Runtime.ImportModuleError: Error: Cannot find module './xhr-sync-worker.js'","Require stack:","- /var/task/index.js","- /var/runtime/index.mjs","    at _loadUserApp (file:///var/runtime/index.mjs:1087:17)","    at async UserFunction.js.module.exports.load (file:///var/runtime/index.mjs:1119:21)","    at async start (file:///var/runtime/index.mjs:1282:23)","    at async file:///var/runtime/index.mjs:1288:1"]}
```

Reply:
Ben just ran into this with a serverless app as well. But unfortunately I couldn't do what he did since this is cdk.

Reply:
This feels like something we should solve with bundling, not patching?